src/SingleCellAnnotator.imjoy.html

```diff
@@ -724,7 +715,7 @@
         api.alert('Please draw a cell mask before saving!');
         return false
       }
-      if(routeFeatures.features.length>1){
+      if((routeFeatures.features.length>1)&&(this.current_task_name!=='segmentation')){
```

Comment:
Instead of checking the task name here, I think it's better to set a global state `single_cell_mode=true/false`, and use that to change the behaviour.